From 038fbc00d6468c8087ff67b3946d098dd888a219 Mon Sep 17 00:00:00 2001
From: Daniel Matuschek <info@crazy-audio.com>
Date: Thu, 16 Jan 2014 07:27:28 +0100
Subject: [PATCH 159/174] BCM2708: Added HiFiBerry Digi configuration option It
 will be compiled as a module by default. This also includes the WM8804
 driver.

Signed-off-by: Daniel Matuschek <daniel@matuschek.net>
---
 arch/arm/configs/bcmrpi_defconfig | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -737,8 +737,10 @@ CONFIG_SND_USB_CAIAQ=m
 CONFIG_SND_USB_CAIAQ_INPUT=y
 CONFIG_SND_USB_6FIRE=m
 CONFIG_SND_SOC=m
+CONFIG_SND_SOC_WM8804=m
 CONFIG_SND_BCM2708_SOC_I2S=m
 CONFIG_SND_BCM2708_SOC_HIFIBERRY_DAC=m
+CONFIG_SND_BCM2708_SOC_HIFIBERRY_DIGI=m
 CONFIG_SND_BCM2708_SOC_RPI_DAC=m
 CONFIG_SOUND_PRIME=m
 CONFIG_HIDRAW=y
